controller/api/apis.go

```diff
@@ -3625,3 +3632,30 @@ type REST_SigstoreRootOfTrustCollection struct {
 type REST_SigstoreVerifierCollection struct {
 	Verifiers []REST_SigstoreVerifier `json:"verifiers"`
 }
+
+type REST_RemoteExportRepository_POST struct {
```

Comment:
1. Which field is for "git service type field" ?
2. What's the difference between "Alias" & "RepositoryName" ? If user can specify "RepositoryName", what's the purpose of "Alias"? Which function reads/writes "Alias" field in this PR?

Reply:
I have implemented 1 and 2 in recent commits.

Yes, the `alias` (now `nickname`) is the NV identifier, not the _actual_ repository name as it appears in github.